### ISSUE TITLE
build fail on "fullTemplateTypeCheck": true

### ISSUE THREAD

**Issue by sandangel** [swimlane/ngx-charts#711]
**I'm submitting a ...**  (check one with "x")
```
- [x] bug report => search github for a similar issue or PR before submitting
[ ] feature request
[ ] support request => Please do not submit support request here
```

**Current behavior**

add `"fullTemplateTypeCheck": true` to angularCompilerOptions of tsconfig.app.json

import ChartCommonModule, LineChartModule to app.module.ts

use example line chart

run `ng build --prod`

```sh
ERROR in node_modules/@swimlane/ngx-charts/release/common/tooltip/tooltip.component.d.ts.TooltipContentComponent.html(12,13): : Property 'context' does not exist on type 'TooltipContentComponent'.
node_modules/@swimlane/ngx-charts/release/common/tooltip/tooltip.component.d.ts.TooltipContentComponent.html(11,13): : Property 'template' does not exist on type 'TooltipContentComponent'.
node_modules/@swimlane/ngx-charts/release/pie-chart/pie-grid-series.component.d.ts.PieGridSeriesComponent.html(12,9): : Property 'max' does not exist on type 'PieGridSeriesComponent'.
```

**Expected behavior**

No error occur

**Reproduction of the problem**

**What is the motivation / use case for changing the behavior?**

**Please tell us about your environment:**

Ubuntu 16.04

* **ngx-charts version:** 7.0.1

* **Angular version:** 5.2.2

* **Browser:** [all | Chrome XX | Firefox XX | IE XX | Safari XX | Mobile Chrome XX | Android X.X Web Browser | iOS XX Safari | iOS XX UIWebView | iOS XX WKWebView ]

* **Language:** [all | TypeScript X.X | ES6/7 | ES5]

**Comment by emoralesb05** [comment]
Second this.. i cant enable `templateTypeCheck` because it complains.

**Comment by Enngage** [comment]
Same here

**Comment by marjan-georgiev** [comment]
Released in 7.1.1
